### DISCUSSION TITLE
How to change page view

### DISCUSSION THREAD

**Discussion by AugustoMN** [webmin/authentic-theme#1683] [Q&A]
Hi, how to change page view from two columns to one column/list?

- Two columns in General defaults
![two-columns-view](https://github.com/webmin/authentic-theme/assets/49379121/6adf2a95-fd84-463a-a748-90aefe94f8f1)

- One column/List in General defaults
![list-view](https://github.com/webmin/authentic-theme/assets/49379121/490113fc-aeca-41d3-9047-9ee6e1d08970)

**Comment by iliaross** [comment]
Hello,

<img width="1179" alt="image" src="https://github.com/webmin/authentic-theme/assets/4426533/65794091-c104-41ca-8602-813f3c88158d">


<img width="1179" alt="image" src="https://github.com/webmin/authentic-theme/assets/4426533/03ec9c3b-0f9e-4c29-943f-0a019da670e1">
